l10n_it_reverse_charge/models/account_move.py

```diff
@@ -484,6 +484,8 @@ def generate_supplier_self_invoice(self):
     def action_post(self):
         ret = super().action_post()
         for invoice in self:
+            if not invoice.is_purchase_document(include_receipts=True):
+                continue
```

Comment:
Da questo capisco che l'autofattura va fatta solo per i documenti di acquisto, è davvero così? Come mai finora non è stato un problema? 
Poi cosa c'entrano tutti i discorsi sugli estratti conto in descrizione? 

Si può chiarire un po', magari con un commento?

Reply:
> è davvero così?

Certo, in quali altri casi?

> Come mai finora non è stato un problema?

Probabilmente perchè nessuno ha registrato righe di estratto conto con partner con posizione fiscale

> Poi cosa c'entrano tutti i discorsi sugli estratti conto in descrizione?

Descrivono lo scenario in cui capita